extensions/blocks/subscriptions/edit.js

```diff
@@ -22,34 +24,30 @@ class SubscriptionEdit extends Component {
 	}
 
 	render() {
-		const { attributes, className, isSelected, setAttributes } = this.props;
+		const { attributes, className, setAttributes } = this.props;
 		const { subscribePlaceholder, showSubscribersTotal } = attributes;
 
-		if ( isSelected ) {
-			return (
-				<div className={ className } role="form">
-					<ToggleControl
-						label={ __( 'Show total subscribers', 'jetpack' ) }
-						checked={ showSubscribersTotal }
-						onChange={ () => {
-							setAttributes( { showSubscribersTotal: ! showSubscribersTotal } );
-						} }
-					/>
-					<TextControl
-						placeholder={ subscribePlaceholder }
-						disabled={ true }
-						onChange={ () => {} }
-					/>
-					<SubmitButton { ...this.props } />
-				</div>
-			);
-		}
-
 		return (
 			<div className={ className } role="form">
+				<InspectorControls>
```

Comment:
`InspectorControls` should ideally be outside the "form" wrapper — can use a React fragment to wrap it all.